OlinkAnalyze/R/Read_NPX_data.R

```diff
@@ -193,6 +193,50 @@ read_NPX_explore <- function(filename) {
                              "QC.Deviation.Inc.Ctrl",
                              "QC.Deviation.Det.Ctrl",
                              "Olink.NPX.Signature.Version"),
+    "header_quant_npxs2.0" = c(header_base,
+                               "Product",
+                               "PanelVersion",
+                               "WellID",
+                               "SampleType",
+                               "Ct",
+                               "LODNPX",
+                               "BelowLOD",
+                               "QuantifiedValue",
+                               "Unit",
+                               "LODQuant",
+                               "LLOQ",
+                               "LQL",
+                               "BelowLQL",
+                               "ULOQ",
+                               "AboveULOQ",
+                               "SampleQC",
+                               "AssayQC",
+                               "MissingFreq",
+                               "Normalization",
+                               "SoftwareVersion",
+                               "SoftwareName",
+                               "AssayType",
```

Comment:
AssayType, QCDevDetCtrl, and QCDevIncCtrl are specific to the extended file and can be removed.